### PR TITLE
API-12749 Clean functionality removed.

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**
Cleaned up clean parameters. Functionality removed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,13 +19,10 @@ It also provides tools for building reproducible (MITRE) databases filled with s
 
 ##### Repopulating the Database
 
-An existing database can be repopulated through one of two methods:
-1. Update
+An existing database can be repopulated through the following method:
+* Update
```
Comment (jblefkove-va) [inline README.md:23]: Combine these? `An existing database can be updated...`

File: run-local.sh [hunk 2]
```diff
@@ -23,11 +22,9 @@ exit 1
 main() {
   case "$1" in
     help|[-]*help) usage "I cant even with this...";;
-    start) createDatabase; shift; break;;
+    clean|start) createDatabase; shift; break;;
```
Comment (jhulbert-va) [inline run-local.sh:25]: At this point, is it even worth including commands? Seems like `./run-local.sh` should just check if a `dqdb` container exists, remove it if it does, then create the database.

Comment (jblefkove-va) [reply run-local.sh:25]: Good point. Back when it took 40+ minutes to populate a full synthetic database, `run-local.sh` was occasionally useful. But it's probably superfluous now.

Comment (jblefkove-va) [reply run-local.sh:25]: On further thought, I think there is value in leaving `run-local.sh` with no arguments for now. When a new migration is applied in qa/staging-lab/lab, it modifies a live, populated database. It is sometimes useful to do that locally, vs always creating the database from scratch.

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**